scripts/player.gd

```diff
@@ -0,0 +1,39 @@
+extends KinematicBody2D
+var vect = Vector2(0,0)
+var up = Vector2(0,-1)
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _physics_process(delta):
+		
+	if Input.is_action_pressed('d'):
```

Comment:
rename these to the actual name of the direction ur moving at for the key mapping